### PR TITLE
Additional tests for the box plugin

### PR DESCRIPTION
**Pull request by AsterMiha**

### CODE REVIEW

File: tests/440-box-plugin.ts [hunk 1]
```diff
@@ -77,21 +77,71 @@ Scenario(
...
+  I.click('$plugin-text-editor', '$plugin-article-content')
+  I.type('/')
+  I.type('Box')
+  I.pressKey('Enter')
+  I.seeElement('$plugin-box-initial-type-chooser')
+
+  I.say('Use the Tab key to select a type')
+  I.pressKey('Tab')
+  I.pressKey('Enter')
+  I.seeElement('$plugin-box')
```
Comment (LarsTheGlidingSquirrel) [inline tests/440-box-plugin.ts:114]: `plugin-box-initial-type-chooser` needs to disappear. `plugin-box` is always visible when there is a box plugin.  

```suggestion
  I.seeElement('$plugin-box')
  I.dontSeeElement('$plugin-box-initial-type-chooser')
```

**Review by LarsTheGlidingSquirrel — CHANGES_REQUESTED** [review]
Thanks Miha for adding more tests! :)
Added some comments. Do you have additional test in mind that we should write?

File: tests/440-box-plugin.ts [hunk 2]
```diff
@@ -77,21 +77,71 @@ Scenario(
...
+Scenario('Change box type using keyboard', async ({ I }) => {
+  I.amOnPage('/entity/create/Article/1377')
+  addBoxPlugin(I, 'blank')
+
+  I.say('Pick a new box type by only using keys')
+  I.pressKey('Tab')
+  I.pressKey('Space')
+  I.pressKey('ArrowDown')
+  I.pressKey('Enter')
+  I.see('Beispiel')
```
Comment (LarsTheGlidingSquirrel) [inline tests/440-box-plugin.ts:127]: Searching for "Beispiel" across the whole page could lead to false positives if "Beispiel" appears somewhere else on the page. But something like this should make it more robust: 
```suggestion
  I.see('Beispiel', '$plugin-box')
```

Comment (hejtful) [reply tests/440-box-plugin.ts:127]: Unfortunately, even this is not specific enough and gives false positives.

This works: `I.see('Beispiel', locate('span.text-brand').inside('$plugin-box'))`

**Review by hejtful — APPROVED** [review]

**Review by LarsTheGlidingSquirrel — APPROVED** [review]
Looks good! Thank you!

**Pull request merged**